.github/workflows/update-intellij-version.yml

```diff
@@ -0,0 +1,47 @@
+name: Update IntelliJ Version
+
+on:
+  pull_request:
+
+jobs:
+  bump-intellij-version:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2.3.4
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+      - name: Update default IntelliJ version
+        run: |
+          sbt "update-intellij / run --java-home /usr/local/openjdk-11"
+          git config user.name github-actions
+          git config user.email github-actions@github.com
+          git add .
+          if [[ -n $(git status | grep 'modified') ]]
+          then
+            RELEASE_LINE=$(grep 'release = \"' core/driver/sources/src/main/resources/reference.conf)
+            INTELLIJ_VERSION=$(awk -F'\"|\"' '{print $2}' <<< "$RELEASE_LINE")
+            git commit -m "updated default IntelliJ version to $INTELLIJ_VERSION"
+            COMMIT_SHA=$(git rev-parse HEAD)
+            git fetch
+            git checkout $GITHUB_BASE_REF
+            curl -fsSL https://cli.github.com/packages/githubcli-archive-keyring.gpg | dd of=/usr/share/keyrings/githubcli-archive-keyring.gpg
+            echo "deb [arch=$(dpkg --print-architecture) signed-by=/usr/share/keyrings/githubcli-archive-keyring.gpg] \
+              https://cli.github.com/packages stable main" | tee /etc/apt/sources.list.d/github-cli.list > /dev/null
+            apt update
+            apt install gh -y
+            gh repo fork --remote=true
+            BRANCH_NAME="update_intellij_version_to_$INTELLIJ_VERSION"
+            git branch $BRANCH_NAME
+            git checkout $BRANCH_NAME
+            git cherry-pick $COMMIT_SHA
```

Comment:
Hmm cherry-pick? 🍒 ⛏️   can the changes be just committed on the right branch from the start?  🤔